draft-foudil-securitytxt.md

```diff
@@ -499,31 +499,31 @@ the "security.txt" file as well or setup a redirect to their own site.
 This can result in security reports not being received by the organization
 or sent to the attacker.
 
-To protect against this, organizations SHOULD digitally sign their "security.txt"
-files (as per {{signature}}), use the canonical directive to sign the location
+To protect against this, organizations should digitally sign their "security.txt"
+files (as per {{signature}}), use the "Canonical" field to sign the locations
 of the file (as per {{canonical}}), and regularly monitor the file and
 the referenced resources to detect tampering.
 
-Security researchers SHOULD check the "security.txt" file including verifying
+Security researchers should triage the "security.txt" file including verifying
 the digital signature and checking any available historical records before using the information
 contained in the file. If "security.txt" file looks suspicious or compromised,
```

Comment:
(Not introduced by this PR, but probably good to fix this typo). One of--
```suggestion
 contained in the file. If a "security.txt" file looks suspicious or compromised,
```
```suggestion
 contained in the file. If the "security.txt" file looks suspicious or compromised,
```

Reply:
will do